src/app/game/game.component.html

```diff
@@ -30,8 +33,12 @@
         <mat-grid-list cols="10" rowHeight="1:1">
           <div *ngFor="let square of shootMap">
             <mat-grid-tile>
-              <button mat-mini-fab color="" [ngClass]="{'button-miss' : square.status==2, 'button-hit': square.status==1, 'button-blocked' : square.status==4}" 
-                                              (click)="status = changeStatus(square.id)">
+              <button mat-mini-fab color="" 
```

Comment:
Change needed to reflect introduced enum

Reply:
separated task created for that issue